### DISCUSSION TITLE
Running in the background

### DISCUSSION THREAD

**Discussion by kiatchun** [ThioJoe/YT-Spammer-Purge#198] [Ideas]
Is it possible to make the app run in the background 24/7 scanning your vids over and over again, so the spammers get deleted immediately?

**Comment by Kurdakov** [comment]
it's possible in this way: write config such that program  requires minimum interactions: just runs and exits (possibly you'd need some minor modifications like automatic exit), now, put it on some cheap web hosting (maybe there are other options, but I know webhostings where you can manage your linux installations remotely will work)  and setup there either cron or systemd service to run script over time. 
watch for quota though, on channel I moderate - due to number of comments the script might be run on one video no more than 30 times per day

**Comment by ThioJoe** [comment]
It should be possible as is with the config file. The main limit would be your API quota
